### ISSUE TITLE
Resolve linting errors

### ISSUE THREAD

**Issue by rexagod** [publiclab/simple-data-grapher#50]
Hi, this is a [first-timers-only issue](https://publiclab.github.io/community-toolbox/#r=all). This means we've worked to make it more legible to folks who either **haven't contributed to our codebase before, or even folks who haven't contributed to open source before**. 

If that's you, we're interested in helping you take the first step and can answer questions and help you out as you do. Note that we're especially interested in contributions from people from groups underrepresented in free and open source software!

We know that the process of creating a pull request is the biggest barrier for new contributors. This issue is for you 💝

If you have contributed before, **consider leaving this one for someone new**, and looking through our general [help wanted](https://github.com/publiclab/plots2/labels/help-wanted) issues. Thanks!

### 🤔 What you will need to know.

Nothing. This issue is meant to welcome you to Open Source :) We are happy to walk you through the process.

### 📋Solution

- [ ] 🙋 **Claim this issue**: Comment below. If someone else has claimed it, ask if they've opened a pull request already and if they're stuck -- maybe you can help them solve a problem or move it along!

- [ ] 📝 **Update**
```
// install ESLint
$ npm i -g eslint

// navigate and run linter
$ cd simple-data-grapher && eslint --fix .

// format code to fix linting errors
```
After the errors are fixed, proceed as specified below.

- [ ] 💾 **Commit** your changes

- [ ] 🔀 **Start a Pull Request**. There are two ways how you can start a pull request:

1. If you are familiar with the terminal or would like to learn it, [here is a great tutorial](https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github) on how to send a pull request using the terminal.

2. You can also [edit files directly in your browser](https://help.github.com/articles/editing-files-in-your-repository/) and open a pull request from there. 

- [ ] 🏁 **Done** Ask in comments for a review :)

### 🤔❓ Questions?

Leave a comment below!

### Is someone else already working on this?

We encourage you to link to this issue by mentioning the issue # in your pull request, so we can see if someone's already started on it. **If someone seem stuck, offer them some help!** Otherwise, [take a look at some other issues you can help with](https://publiclab.github.io/community-toolbox/#r=all). Thanks!

**Comment by IshaGupta18** [comment]
This is great @rexagod ! We'll open a couple more of these!

**Comment by osbornetunde** [comment]
I claim this

**Comment by rexagod** [comment]
@osbornetunde Go ahead, and let us know if you face any problems whatsoever. Thanks!

**Comment by IshaGupta18** [comment]
Yes! When you clone the repo, do npm install to install the dependencies,
and do npm run build, to see the changes in the main file. We'll be
updating the README.md soon. Thanks a lot!n Fri, Jun 21, 2019, 1:00 AM

**Comment by pherwanikar** [comment]
Hey can you  add a WIP label to tickets that have already been picked up to make it clear that they are taken?

**Comment by hemang249** [comment]
Still Open for Solving ? Would like to have a go at this one

**Comment by IshaGupta18** [comment]
I think you can give it a try!

**Comment by dsuarezlogans** [comment]
Hi @IshaGupta18, is this issue available to work on it?

**Comment by raksh07** [comment]
Hi can I try this issue

**Comment by IshaGupta18** [comment]
Sure!

**Comment by alexyiangou** [comment]
Hey! I would like to lend a hand on this as well! :slightly_smiling_face:

**Comment by ovpv** [comment]
Hi, Can i try?

**Comment by IshaGupta18** [comment]
Sure!
